src/utils/data/countries.tsx

```diff
@@ -0,0 +1,4 @@
+export interface Country {
```

Comment:
think about creation of separate file with common `interface`s, `type`s and `enum`s. it's better to create one file for one `interface`s/`type`s/`enum`s